docs/configuration.asciidoc

```diff
@@ -692,6 +692,18 @@ The timeout is applied once the agent has sent the entire request body to the AP
 If the response from the server takes longer than allowed by this timeout,
 the HTTP request is terminated and the TCP socket closed.
 
+[[sanitize-field-names]]
+==== `sanitizeFieldNames`
+* *Type:* Array
+* *Default:* ['password', 'passwd', 'pwd', 'secret', '\*key', '*token*', '\*session*', '\*credit*', '\*card*', 'authorization', 'set-cookie']
+* *Env:* `ELASTIC_SANITIZE_FIELD_NAMES`
+
+Sometimes it is necessary to sanitize the data sent to Elastic APM, e.g. remove sensitive data.
```

Comment:
When possible, use something like `for example` instead of `e.g.`. If you want to use `e.g.`, a comma should follow. In this case, it sounds like we're saying "in other words", so we should swap this to id est (i.e).
As I type all of this, the sentence still doesn't feel right though. Sanitize is in the name of the field, so we can probably get away with something like this:

```suggestion
Remove sensitive data sent to Elastic APM.
```

Reply:
@bmorelli25 _nod_ thanks for catching this, I will change it.  FWIW -- I lifted this language [from some of the other agent docs](https://www.google.com/search?client=firefox-b-1-d&ei=oFHZX_6hOuTi9APh_YzQBA&q=site%3Awww.elastic.co+%22e.g.+remove+sensitive+data%22&oq=site%3Awww.elastic.co+%22e.g.+remove+sensitive+data%22&gs_lcp=CgZwc3ktYWIQAzoECAAQR1CKXlivhwFg-4gBaAJwAngAgAFliAGgC5IBBDIwLjGYAQCgAQGqAQdnd3Mtd2l6yAEIwAEB&sclient=psy-ab&ved=0ahUKEwj-8biunNHtAhVkMX0KHeE-A0oQ4dUDCAw&uact=5) -- if you're looking for a break from React land it's probably worth chasing those down.